libs/dh/admin/feature-invite-user-modal/src/lib/dh-invite-user-modal.component.scss

```
@@ -53,4 +53,23 @@
     display: block;
     margin-bottom: var(--watt-space-ml);
   }
+
+  .info-section {
+    display: flex;
+    flex-direction: row;
+    align-items: start;
```

suggestion: seeing a warning in the console about this. Let's replace it with

```suggestion
    align-items: flex-start;
```